velox/flag_definitions/CMakeLists.txt

```diff
@@ -11,5 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_flag_definitions SHARED flags.cpp)
+add_library(velox_flag_definitions flags.cpp)
```

Comment:
This should be an  `OBJECT` library; i.e. this should not result in a separate `.a` (STATIC) or `.so` (SHARED) file.

If the type is not specified, the type will be STATIC or SHARED based on the value of BUILD_SHARED_LIBS.
https://cmake.org/cmake/help/latest/command/add_library.html

Reply:
Thanks @funrollloops ! PR updated.